src/java.desktop/share/classes/java/awt/geom/Line2D.java

```diff
@@ -112,15 +112,15 @@ public Float(Point2D p1, Point2D p2) {
          * @since 1.2
          */
         public double getX1() {
-            return (double) x1;
+            return x1;
```

Comment:
What do these changes have to do with the subject of the PR ?

Reply:
Just a tiny clean-up in on of affected files. Do you want this to be reverted?

Reply:
I believe it makes to preserve the cast: the fields are of type `float` and explicit cast hints there's a type conversion.

Reply:
Reverted